src/index.md

```diff
@@ -1,37 +1,152 @@
 ---
 layout: default
----
-
-# Apache Beam
+body_class: body--index
 
-#### Apache Beam provides an advanced unified programming model, allowing you to implement batch and streaming data processing jobs that can run on any execution engine.
+logos:
+- title: APEX
+  image_url: /images/logo_apex.png
+  url: "http://apex.apache.org"
+- title: Flink
+  image_url: /images/logo_flink.png
+  url: "http://flink.apache.org"
+- title: Spark
+  image_url: /images/logo_spark.png
+  url: http://spark.apache.org/
+- title: Google Cloud Dataflow
+  image_url: /images/logo_google_cloud.png
+  url: https://cloud.google.com/dataflow/
 
-Apache Beam is:
-* **UNIFIED** - Use a single programming model for both batch and streaming use cases.
-* **PORTABLE** - Execute pipelines on multiple execution environments, including Apache Apex, Apache Flink, Apache Spark, and Google Cloud Dataflow.
-* **EXTENSIBLE** - Write and share new SDKs, IO connectors, and transformation libraries. 
+pillars:
+- title: Unified
+  body: Use a single programming model for both batch and streaming use cases.
+- title: Portable
+  body: Execute pipelines on multiple execution environments.
+- title: Extensible
+  body: Write and share new SDKs, IO connectors, and transformation libraries.
 
-## Get Started
-
-To use Beam for your data processing tasks, start by reading the [Beam Overview]({{ site.baseurl }}/get-started/beam-overview) and performing the steps in the Quickstart for [Java]({{ site.baseurl }}/get-started/quickstart-java) or [Python]({{ site.baseurl }}/get-started/quickstart-py). Then dive into the [Documentation]({{ site.baseurl }}/documentation/) section for in-depth concepts and reference materials for the Beam model, SDKs, and runners.                                    
+cards:
+- quote: "Beam is the perfect framework giving us a lot of flexibility and great functionality."
```

Comment:
Updated Talend quote proposal: "Apache Beam is now part of the foundation of Talend products... a powerful framework that delivers the flexibility and advanced functionality our customers need."

or, if too long, just the part after ...